backend/src/openarchiefbeheer/destruction/api/viewsets.py

```diff
@@ -249,6 +250,11 @@ def update(self, request, *args, **kwargs):
         return super().update(request, *args, **kwargs)
 
     def perform_destroy(self, instance: DestructionList) -> None:
+        if not instance.all_items_can_be_deleted():
+            raise ValidationError(
```

Comment:
If I raise here the `ZaakArchiefactiedatumInFuture` then it will give a 500 error response, but if I raise the `ValidationError` then it will respond with a 400 status to the backend.

Reply:
Makes sense